Build/Classes/Composer/InstallSitePackage.php

```diff
@@ -1,27 +1,24 @@
 <?php
+declare(strict_types=1);
 namespace Neos\BaseDistribution\Composer;
 
 use Composer\Console\Application;
 use Composer\Script\Event;
 use Symfony\Component\Console\Input\ArrayInput;
 use Symfony\Component\Console\Output\ConsoleOutput;
-use Neos\Splash\DistributionBuilder\Service\PackageService;
-use Neos\Splash\DistributionBuilder\Service\JsonFileService;
-use Neos\Splash\DistributionBuilder\Domain\ValueObjects\PackageRequirement;
 
 /**
- * A comopser post-create-project script to allow a choice of adding the demo site (and possible other things in the future)
+ * A composer post-create-project script to allow a choice of adding the demo site (and possible other things in the future)
```

Comment:
I think `comopser` is an infinitely cuter name, but sure, better let's be correct than cute here ;)

Reply:
I had to laugh about this one, too :)